lms/resources/_js_config.py

```diff
@@ -54,6 +54,8 @@ def config(self):
             "authUrl": self._request.route_url("canvas_api.authorize"),
             # Some debug information, currently used in the Gherkin tests.
             "debug": self._debug(),
+            # Tell the JavaScript code whether we're in "dev" mode.
+            "dev": self._request.registry.settings["dev"],
```

Comment:
Should this be part of the `debug` dict?

Reply:
I don't think it has to do with debugging in particular, so maybe not? Maybe `debug` should be under a `dev` dict, if `debug` is only meant for use in dev?

Reply:
That works too, they seem related. The debug dict doesn't have to be anywhere in particular to work, the contents just need to appear in the page content.

Reply:
I'm gonna leave this where it is. The `dev` setting is for indicating that we're in dev, i.e. `make dev`, as opposed to QA, production, _or running the tests_. The `debug` setting is only used in the tests, not in dev. If anything `debug` should maybe be renamed to `tests`. Unless it one day gets extended to also contain stuff that we use for debugging things in the dev server